### DISCUSSION TITLE
"Did you get that SMS" flow and yes/no button styling (Office hours: Sept 29, 2020)

### DISCUSSION THREAD

**Discussion by ang-twilio** [twilio-labs/paste#803] [Office hours] [answered]
**The specific questions I have are...**

- Suggestions for emphasizing the "did you get that SMS" flow. This may currently happen below the fold for certain screens. I would like to discuss how we can up-level this information. 
- How should the yes/no buttons be styled? Aayush left specific comments about these buttons. Shift to comment mode to see his comment.

**Background:** As part of the test an SMS interaction for SMS onboarding, we want a lightweight way to check if the customer got the test SMS or not. If they did, celebrate and move them to further resources. If not, provide information about common troubleshooting.
**Stage of design:** Prototyping
**Links to mockups or prototype:** https://twilio.invisionapp.com/share/5TYUPRZNQ3P#/432461091_msg-Try-Sms-Send

**Comment by kate-porter** [comment, marked as answer]
Thanks @ang-twilio! I've added comments in Invision with some thoughts, and we can chat more in office hours tomorrow too.

**Reply by corinne-nagel** [reply]
Same!

**Comment by ang-twilio** [comment]
Awesome, thank you both! I have some time before office hours so I'll review your comments and try some of the suggestions out.
